zerver/webhooks/slack_incoming/view.py

```diff
@@ -90,19 +95,6 @@ def api_slack_incoming_webhook(
         body = body.strip()
 
     if body != "":
-        body = replace_formatting(replace_links(body).strip())
+        body = convert_slack_formatting(replace_links(body).strip())
```

Comment:
I don't think I follow what the commit message means by "because of the extra formatting from `render_attachment` and `render_block`"?

Reply:
Concretely, formatted strings with emojis and newlines, where the strings are separated by exactly one character. But, I suppose those aren't exclusively produced by `render_attacment` nor `render_block`. I'll rephrase the commit description.